proposals/all_proposals.md

```diff
@@ -0,0 +1,671 @@
+When jobs that fit into this category become the new normal, people working in these jobs should receive a contemporary standard for basic workers' rights, legal methods or tools to empower them to claim and defend these rights if not already existing.
+If not already part of it this roughly outlined problem could be integrated into DiEMs European
+New Deal. More concrete aspects can however not be outlined in this proposal and require a
+discussion by more specialized participants.
+
+# 6 
+# 02.07.2018 European Autonomous Data Network - 18-07-02
+
+Problem:
+
+Right now Cloud Services are centralized, monopolized and often unsecure. The Internet giants are harvesting our data and a centralized server structure provides a ready-made framework for government surveillance. We are not safe on company servers.
+
+Solution:
+
+A decentralised, anonymous, autonomous, encrypted peer-to-peer-network (as opposed to the Siren Servers described by Lanier ).
+This peer-to-peer has to incorporate the following specifications:
+
+- All data on the network is encrypted (even sites and blogs – decrypted upon access)
+- Serverless and decentralized
+- The network creates and maintains itself automatically
+- Ensures Anonymity
+- Ensures Net Neutrality
+- A proxy node obfuscates the clients' (and vaults) location (= IP address in the current broken Internet)
+- Data is stored on hardware and transferred via bandwidth provided by users (so-called Farmers on so-called Vaults)
+- Allows to implement a micropayment system (e.g. users pay for data storage) — Incentive!
+- individual files are split up into pieces (e.g. 1 MByte) , encrypted and spread out over the network (tech detail)
+- Popular Data is being spread increasingly (to where it is demanded) ensuring a growing availability for e.g. popular websites
+- no person has a complete overview over the network (given a minimum size)
+- The bigger the network becomes, the more secure it will get
+- Older Vaults will earn more credibility by doing Proof of Resource (Bandwidth / CPU-Power / Low Downtime) — (tech detail, could also be solved differently)
+
+Demands to public infrastructure:
+
+- The EU has to set up a European Safe Networking Fund that encourages research on an open-source and easy-to-use solution implementing the specifications mentioned above.
+- The EU provides bootstrap servers and minimum amount of storage capacity (storing encrypted data chunks) to get the network up and running.
+What else?
+The Farmers (users that provide hardware and bandwidth) don’t know what they are storing
+The User accesses the network using a Client (e.g. embedded in a Browser). Every user has a so-called Vault that allows the network to store data on their devices → Farmers
+(Providing storage capacity, computation and bandwidth could be linked to virtual currency earnings).
```

Comment:
Spare a few details that are too detailed you just pretty accurately described the GNUnet/secushare project.  <3  The YBTI proposals for legislation aim at obtaining this kind of Internet instead of the current borken one. Instead of asking money to be spent in making this system we recommend to make it a necessary requirement so that the entire industry MUST work on solving this challenge: the challenge for an ethically acceptable Internet. - lynx